src/extension.ts

```diff
@@ -144,6 +152,20 @@ async function backgroundInit(context: vscode.ExtensionContext) {
   );
 }
 
+async function shouldOpenWelcomePage(context: vscode.ExtensionContext) {
+  if (!(await touchIsJustInstalledFlag(context))) {
+    return false;
+  }
+
+  const binaryState = await getBinaryState();
+
+  return (
+    binaryState?.installationTime &&
+    binaryState.flags?.shouldUseNewHubDesign &&
+    isInTheLastHour(new Date(binaryState?.installationTime))
```

Comment:
i think you can use `binaryState.installationTime` without `?` here (because of the check in line 163)